DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -526,22 +549,6 @@ public void RemoveDelegateFromCourse_does_nothing_if_no_current_progress()
                 .MustNotHaveHappened();
         }
 
-        [Test]
-        public void VerifyAdminUserCanAccessCourse_should_return_null_when_course_does_not_exist()

```

Comment:
This was effectively a duplicate test right?

Reply:
Yeah, the body of the test was identical.